packages/ocaml-variants/ocaml-variants.5.1.1+flambda2/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+version: "5.1.1+flambda2"
+synopsis: "Jane Street compiler variant with Flambda2 backend"
+depends: [
+  "ocaml" {= "5.1.1" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+  "dune-secondary" {="3.8.1"}
+  "menhir-secondary"
+  "conf-autoconf" {build}
+]
+conflict-class: "ocaml-core-compiler"
+flags: compiler
```

Comment:
Just to persist a thought: I initially thought, "should we put `avoid-version` here to stop flambda2 accidentally being activated"? However, the fact of `flambda2` is less than `options` means that it should not be accidentally activated, so I think it's just fine as it is!